### ISSUE TITLE
Filemanager 500 Internal Server Error

### ISSUE THREAD

**Issue by ptman** [mholt/caddy#1250]
### 1. What version of Caddy are you running (`caddy -version`)?

Caddy 0.9.3

### 2. What are you trying to do?

Edit /_fileman/README.md

### 3. What is your entire Caddyfile?
```text
http://*:8080
root init/

markdown /
browse /

filemanager /_fileman {
    show init/
    allow_commands false
    allow_edit true
    allow_new true
}
```

### 4. How did you run Caddy (give the full command and describe the execution environment)?

caddy -log stdout -port 8080

### 5. What did you expect to see?

An editor

### 6. What did you see instead (give full error messages and/or log)?

500 Internal Server Error

I'd like to know more, but I don't get anything on log. How do I get more verbose logging?

### 7. How can someone who is starting from scratch reproduce this behavior as minimally as possible?

caddy release + filemanager with included init/ -directory

**Comment by mholt** [comment]
Hi @ptman , thanks for reporting this. Can you please file this issue at [caddy-filemanager](https://github.com/hacdias/caddy-filemanager)? Thanks.

**Comment by pepa65** [comment]
I'll wait for 0.9.4 on this.

**Comment by mholt** [comment]
What will be in version 0.9.4 that you need?

**Comment by pepa65** [comment]
I have a similar setup where I want to serve pages with markdown, and be able to edit them with filemanager.

**Comment by pepa65** [comment]
And I'm guessing templates also don't work with markdown??

**Comment by mholt** [comment]
I believe this was fixed in filemanager, not in Caddy.

**Comment by pepa65** [comment]
But the Caddy binary will not incorporate this fix until 0.9.4, or is that not how it works??

**Comment by mholt** [comment]
No, the plugins can be updated independently. I've updated the build server fairly recently so it might already have the fix.
